.travis.yml

```diff
@@ -5,6 +5,9 @@ scala:
 
 sudo: false
 
+git:
+  depth: 500 # to make sure we have enough depth to power git describe for sbt-dynver (used for snapshot version numbers)
```

Comment:
Not sure which is better, Alpakka does 

https://github.com/akka/alpakka/blob/b0e99507ffce4e4cef6d4e64cc5222455a178653/.travis.yml#L7-L9

Reply:
Yeah, maybe we don't need the full history. It sounds like 500 commits are plenty.